modules/home/components/ResourcesLanding/ResourcesLanding.tsx

```diff
@@ -6,29 +6,34 @@ SPDX-License-Identifier: AGPL-3.0-or-later
 
 */
 
-import { useState } from 'react';
-import { Button, Box, Flex, Heading, Card, Text, Image } from 'theme-ui';
+import { useState, useEffect } from 'react';
+import { Button, Box, Flex, Heading, Card, Text, Image, useThemeUI } from 'theme-ui';
 import { resources, ResourceColor, ResourceCategory } from './resources';
 import { ExternalLink } from 'modules/app/components/ExternalLink';
 import { fadeIn } from 'lib/keyframes';
 import { alpha } from '@theme-ui/color';
 
 const CategoryButton = ({ label, color, active, onClick }) => {
+  const { theme } = useThemeUI();
+  const [badgeSelectorColor, setBadgeSelectorColor] = useState('#000');
+
+  useEffect(() => {
+    setBadgeSelectorColor(theme.rawColors?.badgeSelector as string);
```

Comment:
if `theme.rawColors?.badgeSelector` is undefined maybe we should pass the default value in